### PR TITLE
Merchant items index #93

### LINKED ISSUE #93: User Story 41, Merchant's Items index page
```
As a merchant
When I visit my dashboard
I see a link to view my own items
When I click that link
My URI route should be "/dashboard/items"
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #97

depends on #120

### PR DESCRIPTION
**Pull request by le3ah**
All tests passing; closes #93

### CODE REVIEW

File: app/controllers/merchants_controller.rb [hunk 1]
```diff
@@ -20,6 +20,10 @@ def show
       not_today_satan
     end
   end
+
+  def items_index
+    
+  end
```
[the remark this answers is not in the document]
Comment (le3ah) [reply app/controllers/merchants_controller.rb]: It was from when I was still trying to figure out how to fix my error messages regarding the action in the controller.  Just forgot to delete it.  It's gone now.  Thanks!
